public/tenants/stern/config.ts

```diff
@@ -1,7 +1,7 @@
 const config = {
   tenantName: 'stern',
   tenantURL: 'baeume.stern.de',
-  languages: ['de', 'en'],
+  languages: ['en', 'de'],
```

Comment:
We might want to keep de as default for Stern

Reply:
In that case we would need to add some countries in `de/country.json` as they only exists in `en/country.json` for now and therefore Stern would show the placeholder value if `de` in the default country an a user has chosen the German language. But I assume the country resources will now be transferred to LingoHub and I can add German translations for the remaining country names there.